### PR TITLE
Fix final datastore modification warnings

### PR DESCRIPTION
**Pull request by wvu**
The ones I'm confident about.

Also moved a couple misnamed modules.

### CODE REVIEW

File: modules/auxiliary/server/pxexploit.rb [hunk 1]
```diff
@@ -52,12 +60,6 @@ def initialize
   end
 
   def run
-    if not datastore['TFTPROOT']
```
Comment (todb-r7) [inline modules/auxiliary/server/pxexploit.rb]: should be `unless` since there's no `else`.

File: modules/exploits/windows/local/pxeexploit.rb [hunk 2]
```diff
@@ -67,12 +72,6 @@ def initialize
   end
 
   def exploit
-    if not datastore['TFTPROOT']
```
Comment (todb-r7) [inline modules/exploits/windows/local/pxeexploit.rb]: Prefer `unless` over `if not` when there's no `else`

File: modules/post/windows/manage/pxexploit.rb [hunk 3]
```diff
@@ -46,9 +50,6 @@ def initialize
   end
 
   def run
-    if not datastore['TFTPROOT']
```
Comment (todb-r7) [inline modules/post/windows/manage/pxexploit.rb]: `unless`

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-framework/commit/02fe57e2a1145b43edc33b47681745f9cf0dcb99

**Pull request merged**